nca/NetworkConfig/NetworkLayer.py

```diff
@@ -418,36 +421,9 @@ def _allowed_xgress_conns_optimized(self, is_ingress, peer_container, res_conns_
         return res_conns
 
 
-class IngressNetworkLayer(NetworkLayer):
+class K8sGatewayLayer(K8sCalicoNetworkLayer):
+    pass
 
-    def _allowed_xgress_conns(self, from_peer, to_peer, is_ingress):
-        all_allowed_conns = ConnectionSet(True)
-        allowed_conns, _, _, captured_res = self.collect_policies_conns(from_peer, to_peer, is_ingress)
-        if captured_res:
-            all_allowed_conns = allowed_conns
-        return PolicyConnections(captured=captured_res, allowed_conns=allowed_conns, denied_conns=ConnectionSet(),
-                                 all_allowed_conns=all_allowed_conns)
 
-    def _allowed_xgress_conns_optimized(self, is_ingress, peer_container, res_conns_filter=PolicyConnectionsFilter()):
-        all_peers_and_ips = peer_container.get_all_peers_group(add_external_ips=True, include_dns_entries=True)
-        all_peers_no_ips = peer_container.get_all_peers_group(add_external_ips=False, include_dns_entries=True)
-        non_captured_conns = None
-        res_conns = self.collect_policies_conns_optimized(is_ingress)
-        if res_conns_filter.calc_all_allowed:
-            res_conns.all_allowed_conns = res_conns.allowed_conns
-            non_captured_peers = all_peers_no_ips - res_conns.captured
-            if non_captured_peers:
-                if is_ingress:
-                    non_captured_conns = ConnectivityProperties.make_conn_props_from_dict({"src_peers": all_peers_and_ips,
-                                                                                           "dst_peers": non_captured_peers})
-                else:
-                    non_captured_conns = ConnectivityProperties.make_conn_props_from_dict({"src_peers": non_captured_peers,
-                                                                                           "dst_peers": all_peers_and_ips})
-                res_conns.all_allowed_conns |= non_captured_conns
-        if non_captured_conns and ExplTracker().is_active():
-            src_peers, dst_peers = ExplTracker().extract_peers(non_captured_conns)
-            ExplTracker().add_default_policy(src_peers,
-                                             dst_peers,
-                                             is_ingress
-                                             )
-        return res_conns
+class IstioGatewayLayer(IstioNetworkLayer):
+    pass
```

Comment:
need some documentation here.. whey do k8s-gateway later have same functionality as the k8s-policies layer? same question for istio policies vs istio gateways?

Reply:
As we agreed, no additional documentation needed here.